tests/integration/test_storage_azure_blob_storage/test.py

```diff
@@ -790,6 +790,27 @@ def test_read_subcolumns(cluster):
     assert res == "42\tcont/test_subcolumns.jsonl\t(42,42)\ttest_subcolumns.jsonl\t42\n"
 
 
+def test_read_subcolumn_time(cluster):
+    node = cluster.instances["node"]
+    storage_account_url = cluster.env_variables["AZURITE_STORAGE_ACCOUNT_URL"]
+    azure_query(
+        node,
+        f"INSERT INTO TABLE FUNCTION azureBlobStorage('{storage_account_url}', 'cont', 'test_subcolumn_time.tsv', "
```

Comment:
I think, it is better to make an easy query here. Just insert one Int

Reply:
Seriously?
Ok.